pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -117,27 +117,45 @@ type ServiceClass struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// BrokerName is the reference to the Broker that provides this service.
+	// BrokerName is the reference to the Broker that provides this
+	// ServiceClass.
+	//
 	// Immutable.
 	BrokerName string `json:"brokerName"`
 
-	// Bindable indicates whether a user can create bindings to an instance of
-	// this service. ServicePlan has an optional field called Bindable which
-	// overrides the value of this field.
-	Bindable      bool          `json:"bindable"`
-	Plans         []ServicePlan `json:"plans"`
-	PlanUpdatable bool          `json:"planUpdatable"` // Do we support this?
+	// Description is a short description of this ServiceClass.
+	Description string `json:"description"`
+
+	// Bindable indicates whether a user can create bindings to an Instance
+	// provisioned from this service. ServicePlan has an optional field called
+	// Bindable which overrides the value of this field.
+	Bindable bool `json:"bindable"`
+
+	// Plans is the list of ServicePlans for this ServiceClass.  All
+	// ServiceClasses have at least one ServicePlan.
+	Plans []ServicePlan `json:"plans"`
+
+	// PlanUpdatable indicates whether instances provisioned from this
+	// ServiceClass may change ServicePlans after being provisioned.
+	PlanUpdatable bool `json:"planUpdatable"`
 
 	// ExternalID is the identity of this object for use with the OSB API.
+	//
 	// Immutable.
 	ExternalID string `json:"externalID"`
 
-	// Description is a short description of the service.
-	Description string `json:"description"`
-
-	// ExternalMetadata fields
+	// ExternalMetadata is a blob of information about the ServiceClass, meant
+	// to be user-facing content and display instructions.  This field may
+	// contain platform-specific conventional values.
 	ExternalMetadata *runtime.RawExtension `json:"externalMetadata, omitempty"`
 
+	// Currently, this field is ALPHA: it may change or disappear at any time
+	// and its data will not be migrated.
+	//
+	// Tags is a list of strings that represent different classification
+	// attributes of the ServiceClass.  These are used in CF in a way similar
+	// to Kubernetes labels, but they currently have no special meaning in
```

Comment:
I think I misunderstood the intent. We're trying to say that these are values not in the kube format, they're in whatever format the thing we're attatching too expects. I'm not sure there is a short way to say that, but this reads better to me.